test/functional/admin/detailed_guides_controller_test/national_applicability_test.rb

```diff
@@ -6,6 +6,8 @@ class NationalApplicabilityTest < ActionController::TestCase
 
     setup do
       login_as create(:writer, organisation: create(:organisation))
+      current_user.permissions << "Preview design system"
```

Comment:
Would this work instead?

```suggestion
      login_as_preview_design_system_user :writer
```